### ISSUE TITLE
Recuse "if necessary"

### ISSUE THREAD

**Issue by jfbastien** [tc39/code-of-conduct-proposal#9]
What does this mean:

> In the event of any conflict of interest a committee member must immediately notify the other members, and recuse themselves if necessary.

?

"if necessary" is... imprecise.

**Comment by leobalter** [comment]
recuse the conflict? We can rephrase it to avoid ambiguity

**Comment by jfbastien** [comment]
Let me rephrase: who decides necessity? We assume the committee is benevolent, but from a reporter's POV that's not necessarily a given. How can I trust the committee's ability to recuse upon conflict if they just recuse "if necessary"?

It's just so handwavy, and we're trying to communicate to reporters: tell us, we want to take care of this, if we're not impartial we'll *really* work hard to restore impartiality.

**Comment by gnarf** [comment]
As the editor of the original document this was copied from, the general intent of the wording here was to be: members who are personally connected to a situation will be required to recuse themselves.
